README.md

```diff
@@ -119,29 +117,14 @@ In turn this enables [Wasm components to be able to import][]
 
 ### Other Module Types
 
-Other module types may define their own host module sources. If no module phase import
-is defined, it will fail during the loading phase.
+Any other host-defined module types may define their own host module sources. If a given module does not define a source representation for it's source, importing it with a "source" phase target fails.
```

Comment:
Suggest:

Host-defined module sources must include `%AbstractModuleSourcePrototype%` in their prototype chain and support the *necessary internal slots for revealing their `toString` tag to a brand check, the underlying module source record, and host data*

Reply:
Thanks, added along these lines.